src/esportsbot/cogs/MusicCog.py

```diff
@@ -829,7 +829,8 @@ async def join_member(member):
     @commands.group(
         name="music",
         help="These are commands used to control the music bot. For more detailed command explanations, "
-        "go to https://github.com/FragSoc/esports-bot#music-bot"
+        "go to https://github.com/FragSoc/esports-bot#music-bot",
+        invoke_without_command=True
```

Comment:
afaik invoke_without_command needs to be false so that the `commands.check()` is run before the command is executed

Reply:
I actually just realised that that would cause the `music channels` commands to have that check applied, so not sure what the best option is